Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -214,9 +214,11 @@ def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetche
         (tuple): The iocs and the last page
     """
     iocs: List[dict] = []
+    if last_found_len is None:
+        last_found_len = PAGE_SIZE
     if not last_found_len:
         last_found_len = total_fetched
-    while last_found_len == PAGE_SIZE and limit and total_fetched < limit:
+    while last_found_len in (PAGE_SIZE, PAGE_SIZE - 1) and limit and total_fetched < limit:
```

Comment:
in what scenario will you get `PAGE_SIZE -  1`?
add a comment please

Reply:
This was found in the user's environment, I'll make sure to add a comment.